src/ode/ode/src/fluid_dynamics/ode_fluid_dynamics.cpp

```diff
@@ -245,8 +245,9 @@ int dFluidDynamicsStep (dWorldID w)
       dMultiplyHelper1_331(rav, R, rav);
 
       //printf("linear speed %g  angular speed %g\n", rlvLength, avLength);
-
+#ifdef ODE_DEBUG
```

Comment:
```suggestion
#ifndef dNODEBUG
```

Reply:
(The PR is not yet ready for review, sorry bad old habit to switch to draft for triggering the tests)